### PR TITLE
[#997] README / Home Page: Describe RepoSense features from users' perspective

### LINKED ISSUE #997: README: Briefly describe RepoSense features from the perspective of users
Currently, we have provided a list of RepoSense's capabilities in the README.
It may be good to add a sentence describing these features from the user's perspective.

*An example*
RepoSense is a contribution analysis tool for Git repositories. **It can help educators keep track of work done in student projects, while developers can use it to generate portfolios displaying their open source contributions.** RepoSense generates a static HTML report including contribution information for each author in a repository. The features of the report include... (rest of README)

### PR DESCRIPTION
**Pull request by anubh-v**
Fixes #997 

```
README: Describe RepoSense features from users' perspective

While the README / Home Page have a list of RepoSense's capabilities,
this might not sufficiently convey the value of using RepoSense.
Let's frame RepoSense's features from the
users' perspective. This will help readers appreciate the product.

Let's also make some minor grammar fixes.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,10 @@
 [![deploy on_netlify](https://img.shields.io/badge/deploy-on_netlify-blue.svg)](https://reposense.netlify.com/)
 [![codecov.io](https://codecov.io/gh/reposense/RepoSense/branch/master/graphs/badge.svg?branch=master)](http://codecov.io/github/reposense/RepoSense?branch=master)
 
-RepoSense is a contribution analysis tool for Git repositories. It generates a static HTML report including contribution information for each author in the repository.
-The features of the report includes:
+RepoSense is a contribution analysis tool for Git repositories. It is particularly useful for those who want a chronological visualisation of the code contributions and commits made to a group of repositories. Specifically, educators can use RepoSense to track the work done in student projects, while team leads may use it to oversee their project teams.
+
+RepoSense generates a static HTML report that includes contribution information for each author in each target repository.
```
Comment (Tejas2805) [inline README.md]: Will **each target user** be more appropriate?

**Comment by Tejas2805** [comment]
@anubh-v Is this PR ready for review?

**Comment by anubh-v** [comment]
@Tejas2805 thanks for looking through this PR. I'll update it shortly.

**Comment by anubh-v** [comment]
Ready for reviews

File: README.md (hunk 1 above)
Comment (anubh-v) [reply README.md]: I think it is a bit too verbose to say `each target user in each target repository`.

File: README.md [hunk 2]
```diff
@@ -5,8 +5,10 @@
 [![deploy on_netlify](https://img.shields.io/badge/deploy-on_netlify-blue.svg)](https://reposense.netlify.com/)
 [![codecov.io](https://codecov.io/gh/reposense/RepoSense/branch/master/graphs/badge.svg?branch=master)](http://codecov.io/github/reposense/RepoSense?branch=master)
 
-RepoSense is a contribution analysis tool for Git repositories. It generates a static HTML report including contribution information for each author in the repository.
-The features of the report includes:
+RepoSense is a contribution analysis tool for Git repositories. It is particularly useful for those who want a chronological visualisation of the code contributions and commits made to a group of repositories. Specifically, educators can use RepoSense to track the work done in student projects, while team leads may use it to oversee their project teams.
```
Comment (jamessspanggg) [inline README.md]: code contribution instead of code contributions?

Comment (anubh-v) [reply README.md]: I think it should be "contributions" since it's plural, right?

Comment (jamessspanggg) [reply README.md]: haha yep you're right

**Comment by anubh-v** [comment]
@fzdy1914 may I get your review on this? Thanks for your time

**Comment by fzdy1914** [comment]
Better ask a review from @eugenepeh  for document work.

File: README.md (hunk 2 above)
Comment (eugenepeh) [reply README.md]: i think we can include more group of users e.g. project managers.

**Comment by damithc** [comment]
> do seek @damithc 's opinion too

No objections to the current version.

**Comment by anubh-v** [comment]
> should we also include a gif to show case some of the value add functionalities?

Sure, I think this is a good idea.
We could use the dashboard from one of the modules to illustrate some features.
I'll add it in within this PR?

**Comment by eugenepeh** [comment]
>Sure, I think this is a good idea.
We could use the dashboard from one of the modules to illustrate some features.
I'll add it in within this PR?

Yes please, can reuse/redo one of the gif from user guide for this.

**Comment by anubh-v** [comment]
@eugenepeh, I have added a GIF showing some useful features. May I get your review?

Here is the GIF that I added:
overview

**Comment by damithc** [comment]
> @eugenepeh, I have added a GIF showing some useful features. May I get your review?

Let's minimize the use of GIFs as they are hard to update incrementally when the UI changes.

**Comment by anubh-v** [comment]
@damithc, thanks for the response.

> Let's minimize the use of GIFs as they are hard to update incrementally when the UI changes.

Ok. I have removed the new GIF. Instead, I reused the images and the single GIF that are already present in the User Guide.

I have used the Tabs feature in Markbind to showcase the images / GIF.

**Comment by anubh-v** [comment]
@fzdy1914 could this PR be merged?
I've addressed @eugenepeh 's comment by including some images / GIF in the Home Page.

**Pull request merged**